openedx/core/djangoapps/demographics/models.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+User = get_user_model()
+
+
+class UserDemographics(TimeStampedModel):
+    """
+    A Users Demographics platform related data in support of the Demographics
+    IDA and features
+    """
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    call_to_action_dismissed = models.BooleanField(default=False)
```

Comment:
Suggestion: In general, I think it's easier to read booleans that don't have a negative in their name. Maybe _show_call_to_action_?